react/src/component/PerfTestDashboard.jsx

```diff
@@ -280,7 +400,35 @@ export default class PerfTestDashboard extends React.Component {
                     }
                 </div>
             }
-        </div>
+            {iosEnergyInfo &&
+                <div>
+                    <h3> iOS Energy report</h3>
+                    {isIosEnergyInfoEmpty ?
+                        <div>
+                            There is something wrong when parsing the iOS energy report data, please check the request param or agent.
+                        </div>
```

Comment:
This can be extracted as a component.

Reply:
We should refine this page in the future. Extracting a little bit of codes here didn't make this UI code better.